ipam/spec.go

```diff
@@ -0,0 +1,21 @@
+package ipam
+
+import "context"
+
+type Interface interface {
+	// Test executes the cluster IPAM test using the configured provider
+	// implementation. The test processes the following steps to
+	// ensure the provider specific operator implements guest cluster IPAM
+	// correctly.
+	//
+	//  - Create guest clusters #1, #2, #3.
+	//	- Wait for guest clusters to be ready.
+	//	- Verify that clusters have distinct subnets.
+	//  - Terminate guest cluster #2 and create guest cluster #4.
+	//  - Wait for guest clusters to be deleted and created.  - Verify that
+	//	  clusters have distinct subnets and created cluster #4 did not receive
+	//	  same subnet that deleted cluster #2 had.
```

Comment:
I am not sure what happened to the format here. 

<img width="217" alt="mha_hart_mah_sole" src="https://user-images.githubusercontent.com/552769/42504662-be64cd64-843c-11e8-9d18-a42db09104d4.png">

Reply:
This is probably result of my way of editing this comment here, Vim automatic creation of new comment lines + `gofmt` which ended with mixed tabs and spaces. Fixing... :)